spec/memfs/file_spec.rb

```diff
@@ -1,4 +1,5 @@
 require 'spec_helper'
+require 'pathname'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.